pkg/agent/openflow/client.go

```diff
@@ -480,6 +488,17 @@ func (c *client) UninstallLoadBalancerServiceFromOutsideFlows(svcIP net.IP, svcP
 	return c.deleteFlows(c.serviceFlowCache, cacheKey)
 }
 
+func (c *client) GetServiceFlowKeys(svcIP net.IP, svcPort uint16, protocol binding.Protocol, endpoints []proxy.Endpoint) []string {
+	cacheKey := fmt.Sprintf("Service_%s_%d_%s", svcIP, svcPort, protocol)
```

Comment:
Maybe we should make this format as a function in `client.go`.

Reply:
Make sense. Will do.